README.md

```diff
@@ -13,13 +13,15 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 - [Installation](#installation)
   - [Prerequisites](#prerequisites)
   - [Installing Minishift](#installing-minishift)
-    - [Installing via Homebrew](#installing-via-homebrew)
+    - [Installing with Homebrew](#installing-with-homebrew)
 - [Quickstart](#quickstart)
   - [Starting Minishift](#starting-minishift)
   - [Deploying a sample application](#deploying-a-sample-application)
   - [Reusing the Docker daemon](#reusing-the-docker-daemon)
 - [Documentation](#documentation)
 - [Known Issues](#known-issues)
+  - [Bugs](#bugs)
+  - [Unsupported features](#unsupported-features)
```

Comment:
> My 2 cents, the Known Issues section need not be a part of the Readme at all, especially now, with 'Bugs' being made into a sub-heading which shows in the TOC.

+1, known issues is ok imo, but really what we try to express are "limitations". Regarding to "bugs" I feel exactly the way. No need to highlight this in this way. Also, in its current form the link does not even point to the issue tracker.